corehq/pillows/pillow_dependencies.yml

```diff
@@ -0,0 +1,4 @@
+# pillow-name
+CasePillow:
```

Comment:
Just FYI this pillow is soon to be removed in favour of `CaseToElasticsearchPillow`